src/pages/tutorial/react/step-5.mdx

```diff
@@ -183,6 +183,8 @@ $ cf login -a https://api.ng.bluemix.net --sso
 
 Deploy app using the `cf push` command. Since `manifest.yml` is in our root directory, we don't need to specify it in the push command. But, if you have multiple manifest files that target different environments, it's good practice to specify the file.
 
+Note that to successfully deploy, you may need to update the region code (ie: `api.[REGION].bluemix.net`) in the above address as per https://developer.ibm.com/answers/questions/166972/no-org-and-space-targeted-use-cf-target-o-org-s-sp/.  
```

Comment:
Please update this line to:

```
_Note: To successfully deploy, you may need to update the region code (e.g. `api.[REGION].bluemix.net`). [Learn more.](https://developer.ibm.com/answers/answers/166990/view.html)_
```

So it has the right formatting and directly links to the answer. Also, can you place this line in similar locations in:

* src/pages/tutorial/angular/step-5.mdx
* src/pages/tutorial/vue/step-5.mdx

Reply:
Good suggestion - done!